python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```diff
@@ -420,12 +420,28 @@ $defs:
             description: IPv4 subnet for Loopback0 allocation.
             type: str
             format: ipv4_cidr
+          loopback_ipv4_address:
+            documentation_options:
+              table: node-type-loopback-vtep-configuration
+            description: |-
+              IPv4 address without mask for Loopback0.
+              When set, it takes precedence over `loopback_ipv4_pool`.
+              Note: AVD does not check for validity of the IPv4 address and does not catch duplicates.
+            type: str
```

Comment:
Maybe it would make sense to add "format: ipv4" here

Reply:
We are not (yet) honoring these formats today in the code so that would not bring much more info. Are you saying it could help people reading the documentation to understand that a mask is not needed?

Reply:
Yes, I think this would help in the documentation, because then it is correctly displayed in the column "Value Restrictions"